deeppavlov/models/preprocessors/dirty_comments_preprocessor.py

```diff
@@ -39,25 +40,37 @@ def __call__(self, batch: List[str], **kwargs) -> List[str]:
             list of preprocessed text samples
         """
         f = [x.lower() for x in batch]
+        f = [re.sub("<\S*>", " ", x) for x in f]
+        f = [re.sub('\s+', ' ', x) for x in f]
 
         f = [x.replace("won't", "will not") for x in f]
         f = [x.replace("can't", "cannot") for x in f]
         f = [x.replace("i'm", "i am") for x in f]
         f = [x.replace(" im ", " i am ") for x in f]
-        f = [x.replace("you're ", "you are ") for x in f]
         f = [x.replace("'re", " are") for x in f]
         f = [x.replace("ain't", "is not") for x in f]
         f = [x.replace("'ll", " will") for x in f]
-        f = [x.replace("'t", " not") for x in f]
+        f = [x.replace("n't", " not") for x in f]
         f = [x.replace("'ve", " have") for x in f]
         f = [x.replace("'s", " is") for x in f]
```

Comment:
This could also be used as someones. Like `Alex's` is not `Alex is`

Reply:
yes, I thought about it but do not understand how to divide it correctly